src/applications/disability-benefits/all-claims/pages/privateMedicalRecords.js

```diff
@@ -78,6 +79,26 @@ export const uiSchema = {
         _.get(DATA_PATHS.hasPrivateRecordsToUpload, data, false),
     },
   ),
+  'view:patientAcknowledgement': {
+    'ui:title': ' ',
+    'ui:help': patientAcknowledgmentText,
+    'ui:options': {
+      expandUnder: 'view:uploadPrivateRecordsQualifier',
+      expandUnderCondition: data =>
```

Comment:
I think you might just be able to specify the qualifying value (`false`) instead of a function. See `expandUnderCondition` [here](https://github.com/usds/us-forms-system/blob/57099b4f533d501046a060099a9b8ff02d935524/docs/building-a-form/about-the-schema-and-uischema-objects.md).

Reply:
Just following up...I just tried it, unsuccessfully. I have a theory that this ```expandUnderCondition``` has to be truthy to work. Just haven't totally proven it yet lol.